vite.config.js

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+//import nodePolyfills from 'rollup-plugin-node-polyfills';
```

Comment:
I tried using this to provide `util`, which the `is-my-json-valid` needs but `vite` "externalizes", and while this provided `util` inside the dev server, it stopped the build command from working. The alternative I used here was to npm install `https://www.npmjs.com/package/util`, which provide's node's util for things that don't have it like browsers, then set it up as an alias here.

Reply:
this seems fine if it works. if we need to we could update to something like `ajv` which seems a little more maintained.

I'm kind of surprised that we use JSON schema at all here or would invalidate endpoint responses with it. We don't appear to do that when _making_ the API calls, but we do that when loading persisted ones from state. This should guard the app state from loading an old and invalid state, but it also clears out former API responses.

This is not something we need to do here in this change, but if `is-my-json-valid` is enough of a headache I could imagine it being okay to skip it in order to get rid of `create-react-app`. we can make a follow-up issue to address the schema validation.